.github/workflows/traffic_action.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
     # Calculates traffic and clones and stores in CSV file
     - name: GitHub traffic
-      uses: sangonzal/repository-traffic-action@v0.1.6
+      uses: sangonzal/repository-traffic-action@v.0.1.6
```

Comment:
What the heck? These are some devious tags :laughing:

Reply:
💯